index.js

```diff
@@ -1,64 +1,60 @@
 'use strict';
 
-var assign = require('object.assign');
+const assign = require('object.assign');
+const { declare } = require('@babel/helper-plugin-utils');
 
-var modules = [require('babel-plugin-transform-es2015-modules-commonjs'), {
-  strictMode: true, // add "use strict"
-  strict: false // this allows __esModule to be exported
-}];
-
-var defaultTargets = {
+const defaultTargets = {
   android: 30,
   chrome: 35,
   edge: 14,
   explorer: 9,
   firefox: 52,
   safari: 8,
-  ucandroid: 1
+  ucandroid: 1,
 };
 
 function buildTargets(options) {
   return assign({}, defaultTargets, options.additionalTargets);
 }
 
-module.exports = function buildAirbnbPreset(context, options) {
-  var transpileTargets = (options && options.targets) || buildTargets(options || {});
+module.exports = declare((api, options) => {
+  const transpileTargets = (options && options.targets) || buildTargets(options || {});
+
+  const debug = (options && typeof options.debug === 'boolean') ? !!options.debug : false;
```

Comment:
Can I remove the falsey check for `options`? [Seems like other `babel` presets don't do this](/babel/babel/blob/5cd1276a27277c495cf4ea81bc429f1746febaad/packages/babel-preset-flow/src/index.js#L4).

Reply:
If `api.assertVersion(7);` passes, `options` is guaranteed to be an object, so you should be good.